src/javascript/app_2/App/Components/Layout/Header/__tests__/install_pwa_button.spec.js

```diff
@@ -0,0 +1,19 @@
+import React                  from 'react';
+import { expect }             from 'chai';
+import { configure, shallow } from 'enzyme';
+import Adapter                from 'enzyme-adapter-react-16';
+import { InstallPWAButton }   from '../install_pwa_button.jsx';
+import Button                 from '../../../Form/button';
+
+configure({ adapter: new Adapter() });
+
+describe('ToggleNotificationsDrawer', () => {
```

Comment:
Is the name correct? it does not match the filename and the component we are testing.

Reply:
you're right, it's a mistake